src/main/sensors/rpm_filter.c

```diff
@@ -154,28 +154,27 @@ float rpmFilterDterm(int axis, float value)
     return applyFilter(dtermFilter, axis, value);
 }
 
-static float motorFrequency[MAX_SUPPORTED_MOTORS];
+FAST_RAM_ZERO_INIT static float motorFrequency[MAX_SUPPORTED_MOTORS];
 
-void rpmFilterUpdate()
+FAST_CODE_NOINLINE void rpmFilterUpdate()
```

Comment:
Not sure that this does what you want - `FAST_CODE_NOINLINE` will effectively put a `noinline` on it, forcing it **not** to be put into ITCM RAM.

Reply:
Unfortunately this is exactly what I needed to do since the function would otherwise overflow ITCM. We had discussed it back then and come to the conclusion that we should do a final review of what does and doesn't go into ITCM close to the 4.0 release.